src/style.css

```
@@ -0,0 +1,173 @@
+html {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 198px;
+  height: 404px;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  height: 134px;
+  width: 160px;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+}
+
+.product__name {
+  margin-top: 40px;
+  margin-bottom: 0;
+
+  width: 166px;
+  height: 36px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.product__code {
+  margin-top: 4px;
+  margin-bottom: 0;
+
+  font-size: 10px;
+  font-weight: 400;
+
+  color: #616070;
+}
+
+.review {
+  margin-top: 16px;
+
+  display: flex;
+  width: 100%;
+}
+
+.rating {
+  margin: 0;
+  display: inline-block;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+
+  margin-right: 4px;
+
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.review__stars--0 .stars__star:nth-child(n + 1),
+.review__stars--1 .stars__star:nth-child(n + 2),
+.review__stars--2 .stars__star:nth-child(n + 3),
+.review__stars--3 .stars__star:nth-child(n + 4),
+.review__stars--4 .stars__star:nth-child(n + 5) {
+  background-image: url(images/star.svg);
+}
+
+.review__number {
+  position: static;
+  width: 53px;
+  height: 16px;
+
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+
+  margin-left: 20px;
+
+  color: #060b35;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.price__text {
+  margin-top: 24px;
+  margin-bottom: 16px;
+
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.price__value {
+  margin-top: 24px;
+  margin-bottom: 16px;
+
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.buy {
+  display: flex;
+
+  background: #00acdc;
+  border-radius: 5px;
+
+  height: 40px;
+  width: 166px;
+
+  text-align: center;
+  justify-content: center;
+}
+
+.buy__button {
+  font-weight: 700;
+  font-size: 14px;
+  text-decoration: none;
+  color: #fff;
+
+  padding: 11px 0;
+}
```

now supplement with styling from `.buy`:
```suggestion
  width: 166px;
  text-align: center;
  box-sizing: border-box;

  background: #00acdc;
  border-radius: 5px;
  line-height: 16px;
  border: 1px solid rgba(0, 0, 0, 0);
}
```